README.md

```diff
@@ -1,136 +1,109 @@
+## Prerequisites
 
-- As a 'first-user' network, to access the data streams and APIs that other users provide, you must first exchange your IP and port configuration with a current user.
+To run IRI, you need the following:
 
-- As a 'friend-to-friend' network, you have the privilege of joining new users into the network through your node
-by adding them to your approved neighbors list — ensuring that you both broadcast to them and also receive their broadcasts.
+- 4GB RAM
+- 64-bit processor
+- A public IP address that's either static or connected to a dynamic DNS service such as [duckdns.org](https://www.duckdns.org)
+- [Java](https://openjdk.java.net/install/)
+- [Maven](https://maven.apache.org/what-is-maven.html)
+- Ports 15600 and 14265 must be open
 
-You can **find neighbors** on the #nodesharing channel of our [[Discord server]](https://discord.iota.org/).
+## Installation
 
-Everyone will be welcoming and very happy to help you get connected.
-If you want to get tokens for your testcase, please just ask in one of the communication channels.
+You can do one of the following:
 
-## Reporting Issues
+- Download the pre-built Java file
+- Compile the Java file yourself
 
-If you'd like to contribute to IRI, report bugs, problems or irregularities with this release,
-please see [CONTRIBUTING.md](CONTRIBUTING.md) for guidelines.
+Or, if you use Docker, you can [run the pre-built Docker image](https://docs.iota.org/docs/node-software/0.1/iri/how-to-guides/install-iri#run-iri-in-a-docker-container)
 
-# Documentation
+### Downloading the pre-built Java file
 
-This page contains basic instructions for setting up an IRI node. You can find the full documentation on:
-- Our [documentation website](https://docs.iota.org/docs/node-software/0.1/iri/introduction/overview)
-- [IRI API reference](https://docs.iota.org/docs/node-software/0.1/iri/references/api-reference)
+After every release, we upload a pre-built Java file onto the [GitHub Releases page](https://github.com/iotaledger/iri/releases).
 
-You can also use one of these great community guides:
-- [IOTA Partners guide](https://iota.partners/)
-- [IRI Playbook](https://iri-playbook.readthedocs.io/en/master/index.html)
+You can simply download the latest `.jar` file.
 
-# Installing
+### Compiling the Java file
 
-The preferred option is that you compile yourself.
-The second option is that you utilize the provided jar, 
-which is released whenever there is a new update here: [Github Releases](https://github.com/iotaledger/iri/releases).
+To compile the Java file yourself, do the following on a Linux operating system:
 
-### Compiling yourself  
-
-Make sure to have Maven and Java 8 installed on your computer.
-
-#### To compile & package
-```
-$ git clone https://github.com/iotaledger/iri
-$ cd iri
-$ mvn clean compile
-$ mvn package
+```bash
+git clone https://github.com/iotaledger/iri
+cd iri
+mvn clean compile
+mvn package
```

Comment:
Can this be changed to `mvn clean package` please?

Reply:
Done 😄

Reply:
It is completely my fault for not being clear

Delete `mvn clean compile`
Only leave `mvn clean package`

It can be a 1-liner